src/components/pages/ProfileList/ProfileCard.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './profileList.scss';
+import { Card, Rate } from 'antd';
+import { ScissorOutlined } from '@ant-design/icons';
+const { Meta } = Card;
+
+function ProfileCard(props) {
+  return (
+    <>
+      <Card
+        className="profile-card front"
+        key={props.item.id}
+        cover={
+          <img
+            className="card-image"
+            src={props.item.avatarUrl}
+            alt={props.item.name}
+          />
+        }
+      >
+        <Meta
+          className="card-data"
+          title={props.item.name}
+          description={'user or groomer'}
+        ></Meta>
+        <br />
+        Rating:
+        <br />
+        <Rate
+          character={<ScissorOutlined />}
+          style={{ 'margin-top': '5%', color: 'black' }}
+          disabled
+          defaultValue={5}
```

Comment:
Does this mean that each user has a default rating of 5?

Reply:
No this was just added for styling purposes and at the time I didn't have access to updated DB info